### ISSUE TITLE
Audit screen reader behavior and keyboard-only nav.

### ISSUE THREAD

**Issue by itsjohncs** [Khan/engblog#17]
I think the behavior will be reasonable as is, but it would be better if the screen reader read the post and then moved to the nav bar afterwards, so I'll want to look into that.

**Comment by itsjohncs** [comment]
I tested this a bit ago and found that the subscription buttons need sr-only text, the flow is as I expected so I'll want to figure out how to rejigger the order in which screen readers go through the page (wrapping the nav bar in an html5 nav element might help screenreaders glob the whole thing together too). Finally, the author images should be aria-hidden, and there should be some sr-only text to say "authored by" or something.

That seemed like the only things that needed to change.
